Sources/Swift/Tools/ViewCapture/SentryGraphicsImageRenderer.swift

```diff
@@ -0,0 +1,88 @@
+#if canImport(UIKit) && !SENTRY_NO_UIKIT
+#if os(iOS) || os(tvOS)
+
+import UIKit
+
+/**
+ * Class similar to the ``UIKit/UIGraphicsImageRenderer`` class, but optimized for Sentry.
+ *
+ * We introduced this class, because the ``UIGraphicsImageRenderer`` caused performance issues due to internal caching mechanisms.
+ * During testing we noticed a significant performance improvement by creating the bitmap context directly.
```

Comment:
`m`: Please elaborate on what exactly the internal caching mechanisms are. Are they iOS specific or do they live in our SDK?

Reply:
I have added a declaration in the PR description at "Reuse the UIGraphicsImageRenderer", but to summarize, it's stated in the Apple Documentation in the last paragraph of the _Overview_ here:

https://developer.apple.com/documentation/uikit/uigraphicsimagerenderer

Reply:
I'll add a reference to the code docs